ting_subsearch_suggestions.module

```diff
@@ -179,44 +184,68 @@ function ting_subsearch_suggestions_cron() {
     }
 
     $url = variable_get('ting_subsearch_kpi_webservice_endpoint', NULL);
-    $params['start_period'] = variable_get('ting_subsearch_kpi_start_period', NULL);
-    $params['end_period'] = variable_get('ting_subsearch_kpi_end_period', NULL);
-    $params['format'] = variable_get('subsearch_kpi_format', NULL);
-    $params['language'] = variable_get('subsearch_kpi_language', NULL);
 
-    $account = variable_get('ting_subsearch_kpi_account', NULL);
-    $password = variable_get('ting_subsearch_kpi_password', NULL);
-
-    if (empty($url) || empty($account) || empty($password)) {
+    if (empty($url)) {
       watchdog('ting_subsearch_opensuggestions', 'KPI: Module not configured properly', array(), WATCHDOG_WARNING);
       return;
     }
 
     try {
-      $client = new HttpClient();
```

Comment:
What was the reason to get rid of the `HttpClient`? Should it also be removed from other places if any?
Do these lines still play a role?
```
use GuzzleHttp\Client as HttpClient;
use GuzzleHttp\Exception\ClientException as GuzzleClientException;
```